crud/src/test/java/com/redhat/lightblue/rest/crud/ITCaseCrudResourceTest.java

```diff
@@ -288,7 +288,7 @@ public void testFirstIntegrationTest() throws IOException, ClassNotFoundExceptio
                 "name:1r,iso3code:1,iso2code:0r",
                 "name:a,iso3code:d,iso2code:d",
                 0l,
-                100l).getEntity().toString();
+                100l,null).getEntity().toString();
```

Comment:
Added

On Fri, Jun 3, 2016 at 9:52 AM, Brandon Vulaj <notifications@github.com>
wrote:

> In
> crud/src/test/java/com/redhat/lightblue/rest/crud/ITCaseCrudResourceTest.java
> <https://github.com/lightblue-platform/lightblue-rest/pull/240#discussion_r65730725>
> :
>
> > @@ -288,7 +288,7 @@ public void testFirstIntegrationTest() throws IOException, ClassNotFoundExceptio
> >                  "name:1r,iso3code:1,iso2code:0r",
> >                  "name:a,iso3code:d,iso2code:d",
> >                  0l,
> > -                100l).getEntity().toString();
> > +                100l,null).getEntity().toString();
>
> Can we add a test that tests for when maxResults != null?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lightblue-platform/lightblue-rest/pull/240/files/19b956106f5fbd3febd189ab6d88706ae4685595#r65730725>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADgDDUz3NWdXGaDdKcHOaqsnjUXWrA-Kks5qIE22gaJpZM4IsvZ->
> .
>